source/manual/dns.html.md

```diff
@@ -15,76 +15,100 @@ GOV.UK is responsible for managing several DNS zones, spanning a number of `*.go
 
-## DNS for non-`gov.uk` domains
+### The `blog.gov.uk` domain
 
-GOV.UK also manages DNS zones for some non-`gov.uk` domains (e.g. `independent-inquiry.uk`).
+This is managed in the Government Digital Service Jisc account.
 
-These should be managed in Terraform, with each domain having its own zone configuration file in [govuk-dns-tf][].
+We have a [wildcard CNAME](https://github.com/alphagov/govuk-dns-tf/blob/8fa490bce3d9272e6df69f4dbcb1c1be1b3f07c2/zones/blog.gov.uk.yaml#L18-L21) delegating all subdomains of `blog.gov.uk` to our provider, DXW.
 
-[govuk-dns-tf]: https://github.com/alphagov/govuk-dns-tf
+### The `campaign.gov.uk` domain
+
+This is managed in the Government Digital Service Jisc account.
+
+We have a [wildcard CNAME](https://github.com/alphagov/govuk-dns-tf/blob/1be5ae58e82fb47f0e42cc6f7c2507b424fa9200/zones/campaign.gov.uk.yaml#L91-L95) delegating all subdomains of `campaign.gov.uk` to our provider, DXW.
+
+There are a handful of campaign subdomains that have their own specific NS records or CNAME, which take precedence over the wildcard ([example](https://github.com/alphagov/govuk-dns-tf/blob/1be5ae58e82fb47f0e42cc6f7c2507b424fa9200/zones/campaign.gov.uk.yaml#L54-L60)).
+
+### The `independent.gov.uk` domain
+
+This domain is currently administered by Cabinet Office, who manage the delegation of subdomains (e.g. to `icai.independent.gov.uk`).
+
+[The management of the domain is being looked at](https://trello.com/c/qNpyVaC5/3228-consolidate-co-vs-non-co-domains-in-govuks-jisc-account) by Platform Security & Reliability.
+
+Relatedly, there are a [number of `independent-*.uk` domains managed by GOV.UK](#other-domains-we-manage).
+
+### The `api.gov.uk` domain
+
+This domain is currently administered by Cabinet Office, but delegated to GOV.UK. GOV.UK itself then delegates subdomains (such as www.api.gov.uk) back to Cabinet Office, as well as other subdomains (such as driver-vehicle-licensing.api.gov.uk) to other organisations.
+
+The `api.gov.uk` domain has [A records](https://github.com/alphagov/govuk-dns-tf/blob/552278f8cb155999185aa307124cbae226ad5da4/zones/api.gov.uk.yaml#L3-L8) pointing to Fastly's virtual IP addresses.
+
+`www.api.gov.uk` is a [CNAME to `co-cddo.github.io.`](https://github.com/alphagov/govuk-dns-tf/blob/552278f8cb155999185aa307124cbae226ad5da4/zones/api.gov.uk.yaml#L26-L29).
+
+The Platform Security & Reliability team are [looking at the future management of the api.gov.uk domain](https://trello.com/c/8aXqoeCN).
+
+## Other domains we manage
+
+GOV.UK manages DNS zones for some non-`gov.uk` domains (e.g. `independent-inquiry.uk`). Another example is `alphagov.co.uk`, which is the old domain name GOV.UK publishing used to live on - we maintain records which point to Bouncer so that these URLs redirect.
+
+Some of these domains are managed by us for legacy reasons. Others are defensively registered variations of domains that are in the GOV.UK proposition.
```

Comment:
Citation needed. I'm assuming we only have domains like `independent-inquiry.uk` because `independent.gov.uk` is in the GOV.UK proposition...?

Reply:
I believe `independent-inquiry.uk` is not a `gov.uk` domain for policy reasons — something to do with it needing to be seen as independent from the government iirc. Definitely talk with Policy and Strategy to understand this before writing about it.

Reply:
Sure - happy to remove this 👍